std/typecons.d

```diff
@@ -6850,24 +6850,32 @@ RefCounted!(T, RefCountedAutoInitialize.no) refCounted(T)(T val)
 {
     static struct File
     {
+        static size_t nDestroyed;
         string name;
         @disable this(this); // not copyable
-        ~this() { name = null; }
+        ~this() { name = null; ++nDestroyed; }
     }
 
+    assert(File.nDestroyed == 0);
```

Comment:
This line is not necessary.

Reply:
I have now removed one of the tests for zero...